src/store/actions/requestPermission.js

```diff
@@ -27,6 +27,7 @@ const ALLOWED = [
 ]
 
 export const requestPermission = async ({ state, dispatch }, { origin, data }) => {
+  await dispatch('requestUnlockWallet')
```

Comment:
I think i had this at some point then removed it. can you just double check if it causes issues by trying some dapps when your wallet is lockeD?

Reply:
Hi checked, but requestUnlockWallet only check. if the wallet is locket if not just return and do anything